test cases/fortran/5 static/meson.build

```diff
@@ -1,6 +1,6 @@
-project('try-static-library', 'fortran')
+project('try-static-library', 'fortran', default_options : ['default_library=static'])
 
-static_hello = static_library('static_hello', 'static_hello.f90')
+static_hello = library('static_hello', 'static_hello.f90')
```

Comment:
What was the point of changing this (and the shared ilb test)? There is no noticeable advantage to doing this via `library` since it is explicitly testing that `static_library` works. It is clearer and more explicit to to just use `static_library`.

Reply:
reverted